init.sh

```diff
@@ -112,7 +131,7 @@ then
     echo "DONE"
 
     echo "===> Starting backup..."
-    exec su -pc "mydumper ${CLI_OPTIONS}" ${USER}
+    sudo -u ${USER} mydumper ${CLI_OPTIONS}
```

Comment:
Is the container properly "stoppable" (i.e. `docker stop container` vs `docker kill container`) with just `sudo`?